package.json

```diff
@@ -95,6 +95,7 @@
     ]
   },
   "jest": {
+    "globalSetup": "./jest-global-setup.js",
```

Comment:
https://stackoverflow.com/questions/56261381/how-do-i-set-a-timezone-in-my-jest-config